src/models/dependency.go

```diff
@@ -0,0 +1,42 @@
+package models
+
+import (
+	"golang.org/x/net/context"
+)
+
+type DependencyCondition int
+
+const (
+	OnFailure DependencyCondition = 1 + iota
```

Comment:
@nagachika I use enumeration which starts from 1 to distinguish uninitialized value.
But this doesn't need to distinguish it. I fixed it.